packages/cli/src/controllers/me.controller.ts

```diff
@@ -76,7 +78,10 @@ export class MeController {
 		await this.externalHooks.run('user.profile.beforeUpdate', [userId, currentEmail, payload]);
 
 		await this.userService.update(userId, payload);
-		const user = await this.userService.findOneOrFail({ where: { id: userId } });
+		const user = await this.userRepository.findOneOrFail({
+			where: { id: userId },
+			relations: ['globalRole'],
```

Comment:
why is `globalRole` needed now?

Reply:
To preserve [original behavior](https://github.com/n8n-io/n8n/pull/8186/files#diff-fc68c8bb858239534f9dd5ca345efdb048aecbbfa7329803c055754530ea2b18L37) from `UserService.findOneOrFail()`.